developer-docs-site/docs/guides/wallet-standard.md

```diff
@@ -111,39 +124,51 @@ type [EntryFunctionPayload](https://github.com/aptos-labs/aptos-core/blob/1bc5fd
 
 ```
 
-### connect(), disconnect(), isConnected()
-It is important that dApps, aren't allow to send requests to the wallet until the user acknowledges that they want to see these requests.
+### Connection APIs
+
+The connection APIs ensure that wallets don't accept requests until the user acknowledges that they want to see the requests.  This keeps
+the user state clean, and prevents the user from unknowingly having prompts.
 
-- `connect()` will prompt the user 
+- `connect()` will prompt the user for a connection
     - return `Promise<AccountInfo>`
 - `disconnect()` allows the user to stop giving access to a dApp and also helps the dApp with state management
     - return `Promise<void>`
-- `isConnected()` able to make requests to the wallet to get current state of connection
+- `isConnected()` tells the dapp if it is able to make requests to the wallet
```

Comment:
is connected is not part of the standard https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/packages/wallet-adapter-core/src/types.ts#L26
it is a nice to have function were not all wallets provide (same as sign transaction) - think we can remove it from here.

Reply:
Removed